### PR TITLE
CARDS-12254: Update Cards V4 Developer Documentation

### PR DESCRIPTION
**Pull request by SSchmidtConcur**
Adding the following infos as requested in https://jira.concur.com/browse/CARDS-12254

- Request payload size limit
- Rate limit
- Transactions are dependent on accounts
- Available data centers

### CODE REVIEW

File: src/api-reference/cards/v4.cards-get-started.markdown [hunk 1]
```diff
@@ -14,14 +14,16 @@ The Cards v4 API exposes the following resources:
 Resource|Description
 ---|---
 Account|Partners can create Accounts via bulk API.
-Transaction|Partners can create Transactions via bulk API.
+Transaction|Partners can create Transactions via bulk API.
```
Comment (freitaglukas) [inline src/api-reference/cards/v4.cards-get-started.markdown:17]: Added space at the end on purpose?

**Review by freitaglukas — APPROVED** [review]

File: src/api-reference/cards/v4.cards-get-started.markdown (hunk 1 above)
Comment (SSchmidtConcur) [reply src/api-reference/cards/v4.cards-get-started.markdown]: Was not on purpose - removed: [4a8e94d](https://github.com/SAP-docs/preview.developer.concur.com/pull/1169/commits/4a8e94d8cbe654d9ad3f1f02573f200199130330)

**Review by rrillig — APPROVED** [review]

**Comment by RoAnnC1** [comment]
Hi @SSchmidtConcur, We can include the transaction per call language that you've added, but we never include rate limits in public documentation. Please delete that language and I'll merge.

**Comment by SSchmidtConcur** [comment]
> Hi @SSchmidtConcur, We can include the transaction per call language that you've added, but we never include rate limits in public documentation. Please delete that language and I'll merge.

Removed with 1a52a20bc833882dad2ba8440e557cd5782fd99d

**Comment by SSchmidtConcur** [comment]
Added limit to schema documentation, too. (due to team feedback)
391917aed00018fb8c71205c1409657dfdec55c8

**Review by RoAnnC1 — APPROVED** [review]
:shipit:

**Pull request merged**